packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -15,6 +15,14 @@ import 'text.dart';
 import 'ticker_provider.dart';
 import 'transitions.dart';
 
+// Examples can assume:
```

Comment:
Can you include some context around this comment?  It is probably obvious to people who work in the framework docs a lot, but for others this feels random and strange.

Perhaps you can explain that some docs in this class have source code examples, and those source code examples assume the existence of `MyWidget`.

Also, is this the correct place such a comment?  Do we do this elsewhere where we add a double slash `//` multiline comment above the class comment?

Reply:
This is a magic invocation defined by https://github.com/flutter/flutter/blob/master/dev/bots/analyze-sample-code.dart. [Other such preambles](https://github.com/flutter/flutter/search?q=%22Examples+can+assume%22&unscoped_q=%22Examples+can+assume%22) are not documented, so I didn't feel I should document this one.